build-tools/packages/build-infrastructure/src/test/data/testRepo/package.json

```diff
@@ -15,6 +15,6 @@
 		"@fluid-tools/build-infrastructure": "link:../../../../../build-infrastructure"
 	},
 	"devDependencies": {
-		"@changesets/cli": "^2.27.9"
+		"@changesets/cli": "^2.27.11"
```

Comment:
Test pnpm workspace for some build-tools tests. cross-spawn is a transitive dependency through this.